spec/services/invoices/one_off_service_spec.rb

```diff
@@ -0,0 +1,209 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Invoices::OneOffService, type: :service do
+  subject(:invoice_service) do
+    described_class.new(customer:, timestamp: timestamp.to_i, fees:, currency:)
+  end
+
+  let(:timestamp) { Time.zone.now.beginning_of_month }
+  let(:organization) { create(:organization) }
+  let(:customer) { create(:customer, organization:) }
+  let(:currency) { 'EUR' }
+  let(:add_on_first) { create(:add_on, organization:) }
+  let(:add_on_second) { create(:add_on, amount_cents: 400, organization:) }
+  let(:fees) do
+    [
+      {
+        add_on_code: add_on_first.code,
+        unit_amount_cents: 1200,
+        units: 2,
+        description: 'desc-123'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.